Assets/HoloToolkit/Input/Scripts/Gaze/GazeManager.cs

```diff
@@ -13,6 +13,11 @@ namespace HoloToolkit.Unity.InputModule
     public class GazeManager : Singleton<GazeManager>, IPointingSource
     {
         /// <summary>
+        /// HitInfo property gives access to information at the object being gazed at, if any.
+        /// </summary>
+        public RaycastHit HitInfo { get; private set; }
```

Comment:
Should this go the other way (removing the other hit references)? This should be requested from FocusManager, that way GazeManager isn't special cased in apps, and motion controllers vs gaze is abstracted away.

Reply:
This was kept to keep backwards compatibility for older projects.